nvflare/app_common/tracking/log_writer_metrics_exchanger.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+from typing import Any
+
+from nvflare.apis.analytix import AnalyticsDataType
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_component import FLComponent
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.metric_exchange.metric_exchanger import MetricExchanger
+from nvflare.app_common.tracking.tracker_types import LogWriterName
+from nvflare.app_common.widgets.streaming import AnalyticsSender
+
+
+class LogWriterForMetricsExchanger(FLComponent, ABC):
```

Comment:
if we have the LogWriter that contains both MetricsExchanger and AnalyticsSender I think we don't need this specific one anymore?

Reply:
Right, this was included accidentally